platform/entanglement/test/services/CopyServiceSpec.js

```diff
@@ -41,19 +41,24 @@ define(
         }
 
         describe("CopyService", function () {
+            var policyService;
+
+            beforeEach(function () {
+                policyService = jasmine.createSpyObj(
+                    'policyService',
+                    ['allow']
+                );
+                policyService.allow.andReturn(true);
```

Comment:
policyService being "global" to this spec and having a default here that is required for other specs creates a lot of cognitive distance between the inputs and the expected outputs.  Especially when this default is frequently overridden.

This seems like a good spot to initialize policyService, but it would help comprehension to only specify the return value of the policy service in places where it is important to the functionality being tested.